Configuration/StandardSequences/python/Reconstruction_Data_cff.py

```diff
@@ -16,25 +16,6 @@
         qTest.cleaningThresholds.append(30.)
         qTest.flags.append('HFDigi')
              
-particleFlowRecHitHF.producers[0].qualityTests.append(
```

Comment:
for some reason I thought that this change was not supposed to change anything as it was redundant.
I see small changes in HF-related objects in both run1 and run2 data.
@bachtis could you please clarify
Thank you.